opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -765,14 +765,18 @@ namespace Opm
              */
             if (vapour_active && liquid_active && rstKeywords["PBPD"] > 0) {
                 rstKeywords["PBPD"] = 0;
-                output.insert("PBUB",
-                        Opm::UnitSystem::measure::pressure,
-                        std::move( sd.getCellData("PBUB") ),
-                        data::TargetType::RESTART_AUXILIARY);
-                output.insert("PDEW",
-                        Opm::UnitSystem::measure::pressure,
-                        std::move( sd.getCellData("PDEW") ),
-                        data::TargetType::RESTART_AUXILIARY);
+                if (sd.hasCellData("PBUB")) {
+                    output.insert("PBUB",
+                                  Opm::UnitSystem::measure::pressure,
+                                  std::move( sd.getCellData("PBUB") ),
+                                  data::TargetType::RESTART_AUXILIARY);
+                }
```

Comment:
I think loggin a problem would be nice here, since the user asks for something and we don't output it.